sequencer/src/persistence/fs.rs

```diff
@@ -1454,40 +1454,140 @@ impl MembershipPersistence for Persistence {
         l1_block: u64,
         events: Vec<(EventKey, StakeTableEvent)>,
     ) -> anyhow::Result<()> {
+        if events.is_empty() {
+            return Ok(());
+        }
+
         let mut inner = self.inner.write().await;
         let dir_path = &inner.stake_table_dir_path();
         let events_dir = dir_path.join("events");
 
         fs::create_dir_all(events_dir.clone()).context("failed to create events dir")?;
+        // Read the last l1 finalized for which events has been stored
+        let last_l1_finalized_path = events_dir.join("last_l1_finalized").with_extension("txt");
```

Comment:
just a comment: I sometimes wonder why we use the `.txt` extension but store binary data.

Reply:
changed it to bin extension, I think that makes more sense